metrics-monitoring/README.adoc

```diff
@@ -63,44 +63,48 @@ As a developer or administrator, you can view metrics in {product} to visualize
 [role="_abstract"]
 {product} supports the following metrics for Kafka instances and topics. In the {product} web console, a subset of these metrics is displayed in the *Dashboard* page of each Kafka instance.
 
-Broker metrics::
+Cluster metrics::
 +
 --
-* `kafka_broker_quota_softlimitbytes`: Maximum amount of broker storage before producers are throttled
+* `kafka_namespace:haproxy_server_bytes_in_total:rate5m`: Number of incoming bytes for the cluster in the last five minutes. This metric represents all the data that producers are sending to topics in the cluster. The maximum incoming byte rate is 30 MB per second.
 
-* `kafka_broker_quota_totalstorageusedbytes`: Total storage used by the broker
+* `kafka_namespace:haproxy_server_bytes_out_total:rate5m`: Number of outgoing bytes for the cluster in the last five minutes. This metric represents all the data that producers are receiving from topics in the cluster. The maximum outgoing byte rate is 30 MB per second.
 
-* `kafka_controller_kafkacontroller_global_partition_count`: Number of partitions for each broker in the instance
+* `kafka_namespace:kafka_server_socket_server_metrics_connection_count:sum`: Number of current client connections to the cluster. Kafka clients use persistent connections to interact with brokers in the cluster. For example, a consumer holds a connection to each broker it is receiving data from and a connection to its group coordinator. The maximum number of active connections is 2000.
 
-* `kafka_controller_kafkacontroller_offline_partitions_count`: Number of partitions without an active leader and that are therefore not writable or readable
+* `kafka_namespace:kafka_server_socket_server_metrics_connection_creation_rate:sum`: Number of client connection creations per second for the cluster. Kafka clients use persistent connections to interact with brokers in the cluster. A constant high number of connection creations might indicate a client issue. The maximum connection creation rate is 100 per second.
 
-* `kafka_namespace:haproxy_server_bytes_in_total:rate5m`: Incoming byte rate for the instance in the last five minutes
+* `kafka_topic:kafka_topic_partitions:count`: Number of topics in the cluster. This does not include internal Kafka topics, such as `__consumer_offsets` and `__transaction_state`. The maximum number of topics is 1000.
 
-* `kafka_namespace:haproxy_server_bytes_out_total:rate5m`: Outgoing byte rate for the instance in the last five minutes
+* `kafka_topic:kafka_topic_partitions:sum`: Number of partitions across all topics in the cluster. This does not include internal Kafka topics, such as `__consumer_offsets` and `__transaction_state`. The maximum number of partitions is 1000.
+--
 
-* `kafka_namespace:kafka_server_socket_server_metrics_connection_count:sum`: Number of client connections to the instance
+Broker metrics::
```

Comment:
Right, I didn't included them because I wondered if it was really necessary. I think it's pretty clear that anything under "brokers" is per broker, under "topics" per topic, etc. So the line just didn't provide any really new info in my mind. However, the thing it does provided is the label bit, but I'm not sure again if that's necessary given that the metrics are there listed, and also there was the issue you pointed out in the "cluster" section that the label is missing. So I don't want to point out to users what the label is for each section and then not point that out for cluster, since it's missing. Seemed odd and would likely lead to questions and look like we just left that out by accident. 

So in light of all that, I thought it would avoid the issues by just nixing those lines. Happy to chat more if you disagree or if I've misunderstood.